smart_campus/app/views.py

```diff
@@ -450,6 +511,46 @@ def add_user_favorite_stations(request):
     )
 
 
+@login_required
+def category_add_page(request):
+    """add the category"""
+    if request.method == 'POST':
+        category_form = StationCategoryForm(request.POST)
+
+        if category_form.is_valid():
+            if request.user.can(Permission.ADMIN):
+                stations = Station.objects.all()
+            elif request.user.can(Permission.EDIT):
+                stations = Station.objects.filter(
+                    owner_group=request.user.group)
+            else:
+                return HttpResponseForbidden()
+
+            category_form.save()
+            context = {
+                'name': category_form.cleaned_data['name'],
+                'station_list': [{
+                        'id': station.id,
+                        'name': station.name,
+                        'image_url': station.primary_image_url
+                    }
+                    for station in stations
+                ],
+                'categories': StationCategory.objects.all()
+            }
+
+            # TODO
+            # add the specified category station list
+            return HttpResponse('Success', status=200)
+            # return render(request, 'station_list.html', context)
```

Comment:
Again, ***DO NOT*** add code that is commented out.

Reply:
okay, removed it.